### ISSUE TITLE
tmux THEN resize window works, resize window THEN tmux does not

### ISSUE THREAD

**Issue by avelicu** [chjj/tty.js#112]
Just trying out tmux on OSX 10.9.2.
It works great except for one strange quirk:

If I open a new terminal (at the default 80x25), start tmux, and then resize or maximize the terminal, everything is fine, as in the first screenshot.

However, if I open a new terminal, resize it and then run tmux, it is not working properly, as in the second screenshot.

I am using tmux 1.8_1 from MacPorts and tty.js from NPM.
Am I doing something wrong? :)
(I know you can see large window numbers in the screenshots, but I have verified this problem with fresh tmux sessions as well)
![tmux-bad](https://f.cloud.github.com/assets/1438808/2320727/5a99ce80-a393-11e3-9f47-9635dd176131.png)
![tmux-good](https://f.cloud.github.com/assets/1438808/2320731/65c62146-a393-11e3-9d07-fbe689a874eb.png)

**Comment by Lakshmipathi** [comment]
@GottZ , Did that fix work for you with tty.js ? I'm still seeing this issue even after apply that fix with tty.js

**Comment by GottZ** [comment]
@Lakshmipathi yup. for me it did the fix. sadly i have not got my homeserver up and running right now so i cannot diff if i have made other changes aswell. i will provide feedback in some days.
